### PR TITLE
[FIX-3]- BE integration (images agregation)

### PR DESCRIPTION
**Pull request by IrynaMukhina**

### CODE REVIEW

File: src/app/components/recently-viewed/recently-viewed.component.ts [hunk 1]
```diff
@@ -39,9 +39,12 @@ export class RecentlyViewedComponent implements OnInit, OnDestroy {
         switchMap(ids => {
           return forkJoin(this.productService.getProductsByIds(ids, ProductFormat.short));
         }),
+        tap((product) => {console.log(product)})
```
Comment (andriiveremchuk) [inline src/app/components/recently-viewed/recently-viewed.component.ts]: please remove console.logs (check everywhere)

File: src/app/components/recently-viewed/recently-viewed.component.ts [hunk 2]
```diff
@@ -38,11 +38,10 @@ export class RecentlyViewedComponent implements OnInit, OnDestroy {
...
         switchMap(ids => {
           return forkJoin(this.productService.getProductsByIds(ids, ProductFormat.short));
-        }),
+        })
       )
-      .subscribe(product => {
-        this.recentlyViewedProducts = product.sort((a, b) => b.order - a.order);
-       });
+      .subscribe(products =>
+        this.recentlyViewedProducts = products.sort((a, b) => b.order - a.order));
```
Comment (TorskyM) [inline src/app/components/recently-viewed/recently-viewed.component.ts]: Set appropriate names for variables

File: src/app/store/reducers/products.reducer.ts [hunk 3]
```diff
@@ -1,19 +1,17 @@
 import {
   SetProducts,  LoadProducts, ProductsActions, FilterByGender, FilterBySeason, SetProductImages
 } from '../actions/products.action';
-import { IProduct, ICloudinaryImage } from '../../interfaces';
+import { IProduct } from '../../interfaces';
 import { URLs } from 'src/app/app.enum';
```
Comment (TorskyM) [inline src/app/store/reducers/products.reducer.ts]: Don't mix url refs

File: src/app/shared/services/product.service.ts [hunk 4]
```diff
@@ -37,20 +30,16 @@ export class ProductService implements OnDestroy {
 
   public setCartItemsToLocalStorage(): void {
     this.store.select(getCartProductItems)
-    .subscribe(products =>
-      localStorage.setItem(this.CART_KEY, JSON.stringify(products))
-      );
-    }
+      .subscribe(products =>
```
Comment (castan1c) [inline src/app/shared/services/product.service.ts]: please, return the AutoUnsubscribe() if u use subscribe method. check everywhere

File: src/app/store/reducers/products.reducer.ts (hunk 3 above)
Comment (IrynaMukhina) [reply src/app/store/reducers/products.reducer.ts]: Any special reason for that?

**Comment by IrynaMukhina** [comment]
thanks for reviewing!!!

**Pull request merged**